source/_integrations/honeywell.markdown

```diff
@@ -16,16 +16,42 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The Honeywell integration integrates Home Assistant with _US-based_ [Honeywell Total Connect Comfort (TCC)](https://mytotalconnectcomfort.com/portal/) climate systems.
+The `Honeywell` integration integrates Home Assistant with _US-based_ [Honeywell Total Connect Comfort (TCC)](https://mytotalconnectcomfort.com/portal/) climate systems.
 
-It uses the [somecomfort](https://github.com/kk7ds/somecomfort) client library. It does not support the home security functionality of TCC.
+It uses the [aiosomecomfort](https://github.com/mkmer/AIOSomecomfort) client library. It does not support the home security functionality of TCC.
 
 If your system is compatible with this integration, then you will be able access it via [https://mytotalconnectcomfort.com/portal/](https://mytotalconnectcomfort.com/portal/) (note the `/portal/`).
 
+- [Supported hardware](#supported-hardware)
+- [Climate](#climate)
+- [Sensor](#sensor)
+
+## Supported hardware
+
+Any US based Honeywell thermostat the uses [Honeywell Total Connect Comfort (TCC)](https://mytotalconnectcomfort.com/portal/)
+
+{% include integrations/config_flow.md %}
+
+## Climate
+
+The `honeywell` climate platform integrates Honeywell US based thermostats into Home Assistant, allowing control of the thermostat through the user interface. The current inside temperature, operating mode, and fan state are also displayed on the thermostat card.
+
+The following services are also available:
+
+- [**set_hvac_mode**](/integrations/climate/#service-climateset_hvac_mode) (`off`, `heat`, `cool`,`auto`)
+- [**target temperature**](/integrations/climate#service-climateset_temperature)
+- [**turn on/off**](/integrations/climate#service-climateturn_on)
+- [**fan mode**](/integrations/climate#service-climateset_fan_mode) (`on`, `auto`, `diffuse`)
+- [**target humidity**](/integrations/climate#service-climateset_humidity) 
+- [**Preset**](/integrations/climate#service-climateset_preset_mode)
+- [**Aux Heat**](/integrations/climate#service-climateset_aux_heat)
```

Comment:
These are services, the current list is a weird combination of service human-readable names and the object IDs of the services. All emphasized in bold.

I suggest either using service names, and their service IDs (`climate.set_hvac_mode`), or just making a single link to the climate page where they can find additional services.

Reply:
I copied this from a different integration - probably should fix that one to.